solr/modules/extraction/src/test/org/apache/solr/handler/extraction/TestXLSXResponseWriter.java

```diff
@@ -38,8 +38,10 @@
 import org.apache.solr.search.SolrReturnFields;
 import org.junit.AfterClass;
 import org.junit.BeforeClass;
+import org.junit.Ignore;
 import org.junit.Test;
 
+@Ignore
```

Comment:
Is this meant to be here as part of the PR< or just to get past a build issue?

Reply:
Just a build fix. Won't commit.